operations/validate_csm_health.md

```diff
@@ -408,54 +166,45 @@ Known failures and how to recover:
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-storage
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s ncn_uptimes
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s node_resource_consumption
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s pods_not_running
 ```
+<a name="known-issues"></a>
+#### 1.2.1 Known Issues
 
-And the Kubernetes test suite via:
+* pods_not_running
+  - If the output of `pods_not_running` indicates that there are pods in the `Evicted` state, it may be due to the /root file system being filled up on the Kubernetes node in question. Kubernetes will begin evicting pods once the root file system space is at 85% until it is back under 80%. This may commonly happen on ncn-m001 as it is a location that install and doc files may be downloaded to. It may be necessary to clean up space in the /root directory if this is the root cause of pod evictions. The following commands can be used to determine if analysis of files under /root is needed to free-up space.
 
-```bash
-pit# /opt/cray/tests/install/ncn/automated/ncn-kubernetes-checks
-```
+    ```bash
+    ncn# df -h /root
+    Filesystem      Size  Used Avail Use% Mounted on
+    LiveOS_rootfs   280G  245G   35G  88% /
+    ```
 
-<a name="autogoss-issues"></a>
-#### 1.8.1 Known Test Issues
+    ```bash
+    ncn# du -h -s /root/
+    225G  /root/
+    ```
 
-* These tests can only reliably be executed from the PIT node. Should be addressed in a future release.
-* K8S Test: Kubernetes Query BSS Cloud-init for ca-certs
-  - May fail immediately after platform install. Should pass after the TrustedCerts Operator has updated BSS (Global cloud-init meta) with CA certificates.
-* K8S Test: Kubernetes Velero No Failed Backups
-  - Because of a [known issue](https://github.com/vmware-tanzu/velero/issues/1980) with Velero, a backup may be attempted immediately upon the deployment of a backup schedule (for example, vault). It may be necessary to use the `velero` command to delete backups from a Kubernetes node to clear this situation.
+    ```bash
+    ncn# du -ah -B 1024M /root | sort -n -r | head -n 10
+    ```
+  - The `cray-crus-` pod is expected to be in the Init state until slurm and munge
+are installed. In particular, this will be the case if executing this as part of the validation after completing the [Install CSM Services](../install/install_csm_services.md).
+If in doubt, validate the CRUS service using the [CMS Validation Tool](#sms-health-checks). If the CRUS check passes using that tool, do not worry about the `cray-crus-` pod state.
+  - The `hmn-discovery` and `cray-dns-unbound-manager` cronjob pods may be in a 'NotReady' state. This is expected as these pods are periodically started and transition to the completed state.
 
 
 <a name="optional-check-of-system-management-monitoring-tools"></a>
-### 1.9 Optional Check of System Management Monitoring Tools
+### 1.3 OPTIONAL Check of System Management Monitoring Tools
```

Comment:
Seems odd that we have to number the sections manually... can't a computer do this for us?

Reply:
I know this is possible and will look into it

Reply:
Given the Topics at the top are all hard coded and include the section numbers, I plan to leave this as is for now.